packages/teams-components/stories/Toolbar/Default.stories.tsx

```diff
@@ -0,0 +1,184 @@
+const SendIcon = bundleIcon(SendFilled, SendRegular);
+const DeviceEqIcon = bundleIcon(DeviceEqFilled, DeviceEqRegular);
+const DocumentEditIcon = bundleIcon(DocumentEditFilled, DocumentEditRegular);
+const MoreHorizontalIcon = bundleIcon(
+  MoreHorizontalFilled,
+  MoreHorizontalRegular
+);
+const DocumentOnePageMultipleIcon = bundleIcon(
+  DocumentOnePageMultipleFilled,
+  DocumentOnePageMultipleRegular
+);
+
+export const Default = () => {
+  return (
+    <>
+      <EditorToolbar />
+      <GroupChatHeader />
+    </>
+  );
+};
+
+export const EditorToolbar = () => {
+  const [toggle, setToggle] = React.useState(false);
+  return (
+    <Toolbar>
+      <ToolbarToggleButton
+        checked={toggle}
+        onClick={() => setToggle((s) => !s)}
+        appearance="transparent"
+        icon={<TextEditStyleIcon />}
+        title="Show formatting options"
+      />
+      <Menu>
+        <MenuTrigger>
+          <ToolbarMenuButton
+            appearance="transparent"
+            icon={<EmojiIcon />}
+            title="Emojis, GIFs and Stickers"
+          />
+        </MenuTrigger>
+        <MenuPopover>
+          <MenuList>
+            <MenuItem>Emoji</MenuItem>
+            <MenuItem>GIF</MenuItem>
+            <MenuItem>Sticker</MenuItem>
+          </MenuList>
+        </MenuPopover>
+      </Menu>
+      <ToolbarButton
+        appearance="transparent"
+        icon={<PenSparkleIcon />}
+        title="Rewrite with copilot"
+      />
+      <Menu>
+        <MenuTrigger>
+          <ToolbarMenuButton
+            appearance="transparent"
+            icon={<AddIcon />}
+            title="Actions and apps"
+          />
+        </MenuTrigger>
+        <MenuPopover>
+          <MenuList>
+            <MenuItem>Attach file</MenuItem>
+            <MenuItem>Schedule message</MenuItem>
+            <MenuItem>Set delivery options</MenuItem>
+          </MenuList>
+        </MenuPopover>
+      </Menu>
+
+      <ToolbarDivider />
+      <ToolbarButton
+        appearance="transparent"
+        icon={<SendIcon />}
+        title="Send message"
+      />
+    </Toolbar>
+  );
+};
+
+export const GroupChatHeader = () => {
+  const [toggleNotes, setToggleNotes] = React.useState(false);
+  const [toggleAINotes, setToggleAINotes] = React.useState(false);
+  return (
+    <Toolbar>
+      <ToolbarButton appearance="transparent" icon={<DeviceEqIcon />}>
+        Meet now
+      </ToolbarButton>
+      <Menu>
+        <MenuTrigger>
+          <ToolbarMenuButton
+            title="Add people, agents and bots"
+            appearance="transparent"
+            icon={<PeopleAddIcon />}
+          >
+            72
+          </ToolbarMenuButton>
+        </MenuTrigger>
+        <MenuPopover>
+          <MenuList>
+            <MenuItem>Max Mustermann</MenuItem>
+            <MenuItem>Jane Doe</MenuItem>
+            <MenuItem>Pierre Dupont</MenuItem>
+          </MenuList>
+        </MenuPopover>
+      </Menu>
```

Comment:
This one is interesting: what should we do with `Menu`?

Reply:
We have plans for strict menu, resolving this for now since we'll get back to this